src/components/DatePickerWidget/DatePickerWidget.tsx

```diff
@@ -88,12 +88,20 @@ const DatePickerWidget: FC<Props> = ({ id, disableInteractions }) => {
   const [isOpen, setOpen] = useState(false);
   const [dropdown, setDropdown] = useState({ top: 0, left: 0, width: 0 });
   const [localData, setLocalData] = useState(initialData);
+  const MIN_DROPDOWN_WIDTH = 320;
```

Comment:
Please, move it outside of render function - and set it as component property with default value.